resources/zib/ext-StopType.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="ext-StopType" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/ext-StopType" />
+  <name value="ExtStopType" />
+  <title value="extStopType" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="An extension to provide the manner in which medication is discontinued (temporary or definitive)." />
+  <purpose value="This extension represents MedicationAgreementStopType, AdministrationAgreementStopType and MedicationUseStopType of zibs [MedicationAgreement](https://zibs.nl/wiki/MedicationAgreement-v1.2(2020EN)), [AdministrationAgreement](https://zibs.nl/wiki/AdministrationAgreement-v1.0.3(2020EN)) and [MedicationUse2](https://zibs.nl/wiki/MedicationUse2-v1.1.1(2020EN)) respectively." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <kind value="complex-type" />
+  <abstract value="false" />
+  <context>
+    <type value="fhirpath" />
+    <expression value="MedicationRequest" />
+  </context>
+  <context>
+    <type value="fhirpath" />
+    <expression value="MedicationDispense" />
+  </context>
+  <context>
+    <type value="fhirpath" />
+    <expression value="MediationStatement" />
+  </context>
+  <type value="Extension" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Extension" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Extension">
+      <path value="Extension" />
+      <definition value="The MedicationAgreementStopType, AdministrationAgreementStopType and MedicationUseStopType concepts are mapped to this ModifierExtension and not the `.status` element of the respective resources because it has different meaning and function. The `.status` element represents the status of the resource at the moment of requesting the resource. That – in itself – makes mapping the StopType (which may never change in an instance) to status not suitable. For example: a medication agreement which has a StopType, should in itself be superseded by a new medication agreement instance to resume the medication treatment rather then updating the `.status`. There are no specifications for status at the moment, and it is not forbidden to fill the status of the resource." />

```

Comment:
This reasoning makes no sense at all to me. This implies that systems SHOULD make a new zib instance that supersedes the previous one for every mutation -- why would you even include the StopType "Tijdelijk" then? We're not dealing with persisting information, just with the exchange of information -- and this will always be a representation of how it currently is. I don't see any difference between `.status` and the zib concept StopType, at least not according to this description.

Reply:
Created https://github.com/Nictiz/Nictiz-R4-zib2020/issues/114 for this, and proposed a meeting to discuss this mapping/extension. Will move forward with the current design, potentially adjusting the mapping in the future OR provide better documentation.